### PR TITLE
Changeallhead

### PR DESCRIPTION
**Pull request by GaoTianDuo**
Modified the style of the home page:head,introduction,feature.
1、Add icons
2、The style has been modified with reference to the design prototype

### CODE REVIEW

**Comment by GaoTianDuo** [comment]
feature
head
introduction

**Comment by chengshiwen** [comment]
@GaoTianDuo Hi, please resolve the conflicts first, thx~

File: src/pages/home/index.jsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ class Home extends Language {
             </div>
             <img src={getLink(dataSource.introduction.img)} />
           </div>
-        </section>
+        </section>
```
Comment (chengshiwen) [inline src/pages/home/index.jsx:109]: Remove the extra spaces

File: src/pages/docs/index.md.jsx [hunk 2]
```diff
@@ -73,6 +73,7 @@ class Docs extends Md2Html(Language) {
             ref={(node) => { this.markdownContainer = node; }}
             dangerouslySetInnerHTML={{ __html }}
           />
+
```
Comment (chengshiwen) [inline src/pages/docs/index.md.jsx:76]: Remove the extra empty line

File: src/pages/community/index.md.jsx [hunk 3]
```diff
@@ -21,6 +21,7 @@ class Community extends Md2Html(Language) {
           language={language}
           onLanguageChange={this.onLanguageChange}
         />
+
```
Comment (chengshiwen) [inline src/pages/community/index.md.jsx:24]: Remove the extra empty line

File: src/pages/community/index.jsx [hunk 4]
```diff
@@ -24,6 +24,7 @@ class Community extends Language {
           language={language}
           onLanguageChange={this.onLanguageChange}
         />
+
```
Comment (chengshiwen) [inline src/pages/community/index.jsx:27]: Remove the extra empty line

File: src/pages/blog/index.jsx [hunk 5]
```diff
@@ -23,6 +23,7 @@ class Blog extends Language {
           language={language}
           onLanguageChange={this.onLanguageChange}
         />
+
```
Comment (chengshiwen) [inline src/pages/blog/index.jsx:26]: Remove the extra empty line

File: site_config/home.jsx [hunk 6]
```diff
@@ -24,33 +24,36 @@ export default {
     introduction: {
       title: '分布式易扩展的可视化DAG工作流任务调度系统',
       desc: 'Apache DolphinScheduler是一个分布式去中心化，易扩展的可视化DAG工作流任务调度系统。致力于解决数据处理流程中错综复杂的依赖关系，使调度系统在数据处理流程中开箱即用。',
-      img: '/img/architecture.jpg',
+      img: '/img/framework.png',
```
Comment (chengshiwen) [inline site_config/home.jsx:27]: `framework.png` was not added by git

File: site_config/home.jsx [hunk 7]
```diff
@@ -24,33 +24,36 @@ export default {
...
     },
     features: {
       title: '特性一览',
       list: [
         {
-          img: '/img/feature_loadbalances.png',
+          img: '/img/ha.png',
```
Comment (chengshiwen) [inline site_config/home.jsx:33]: `ha.png` was not added by git

File: site_config/home.jsx [hunk 8]
```diff
@@ -24,33 +24,36 @@ export default {
...
           title: '高可靠性',
           content: '去中心化的多Master和多Worker, 自身支持HA功能, 采用任务队列来避免过载，不会造成机器卡死',
         },
         {
-          img: '/img/feature_hogh.png',
+          img: '/img/easyuse.png',
```
Comment (chengshiwen) [inline site_config/home.jsx:38]: `easyuse.png` was not added by git

File: site_config/home.jsx [hunk 9]
```diff
@@ -24,33 +24,36 @@ export default {
...
           title: '简单易用',
           content: 'DAG监控界面，所有流程定义都是可视化，通过拖拽任务定制DAG，通过API方式与第三方系统对接, 一键部署',
         },
         {
-          img: '/img/feature_service.png',
+          img: '/img/scene.png',
```
Comment (chengshiwen) [inline site_config/home.jsx:43]: `scene.png` was not added by git

File: site_config/home.jsx [hunk 10]
```diff
@@ -24,33 +24,36 @@ export default {
...
           title: '丰富的使用场景',
           content: '支持暂停恢复操作. 支持多租户，更好的应对大数据的使用场景. 支持更多的任务类型，如 spark, hive, mr, python, sub_process, shell',
         },
         {
-          img: '/img/feature_runtime.png',
+          img: '/img/scaleout.png',
```
Comment (chengshiwen) [inline site_config/home.jsx:48]: `scaleout.png` was not added by git

**Review by chengshiwen — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! There are a few minor issues that need to be fixed here😊

**Comment by GaoTianDuo** [comment]
add img files and fix the Changeallhead #344.

**Comment by CalvinKirs** [comment]
todo :Screen adaptive

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**